spec.html

```diff
@@ -16385,7 +16390,8 @@ <h2>Syntax</h2>
         `\` UnicodeEscapeSequence
 
       IdentifierStartChar ::
-        UnicodeIDStart
+        AsciiLetter
+        UnicodeIDStart but not AsciiLetter
```

Comment:
Why is this change made? This doesn't seem like an improvement.

Reply:
Mostly to give a reason for |AsciiLetter| appearing a few lines down, but I guess that's not necessary. Reverted.